pkg/nuclide/ui/tree/lib/TreeRootComponent.js

```diff
@@ -65,6 +65,18 @@ function toggleSetHas(
   return added;
 }
 
+function isPathIgnored(path: string): boolean {
```

Comment:
This can be simplified with `some`:

```js
function isPathIgnored(path: string): boolean {
  return atom.project.getRepositories().some(repo => {
    return repo && repo.isPathIgnored(path);
  });
}
```